README.md

```diff
@@ -34,6 +33,45 @@ Special notes:
  * Since we cannot distribute Java Richards, a patch is required to derive the
    tracing version (`patches/trace_java_richards.diff`)
 
+## Using the scripts here to analyse an existing benchmark
+
+### Requirements
+
+  * Python 2.7 (the code here is not Python 3.x ready)
+  * bzip2 / buzip2
+  * Python modules: numpy, rpy2
+  * R (e.g. `sudo apt-get install r-base`)
+  * Python modules required for plotting: matplotlib, seaborn
+  * Required for generating LaTeX tables: pdflatex (e.g. `sudo apt-get install tetex-live`)
+
+### CSV format
+
+Scripts here expect to work with benchmark data in CSV format. The first row
```

Comment:
Let's change the first sentence to something like "The scripts here take CSV files as input. The format is as follows. The first row must ...".